src/fiskaltrust.Launcher/Services/EncodingProvider.cs

```diff
@@ -0,0 +1,22 @@
+using System.Text;
+
+namespace fiskaltrust.Launcher.Services
+{
+  public class LauncherEncodingProvider : EncodingProvider
+  {
+    public override Encoding? GetEncoding(int codepage) => CodePagesEncodingProvider.Instance.GetEncoding(codepage);
+
+    public override Encoding? GetEncoding(string name)
```

Comment:
I've removed the `CodePagesEncodingProvider.Instance.GetEncoding(codepage);` from above because it seems `CodePagesEncodingProvider` is not really the default behaviour and we don't have access to the default (which is why I'm just calling `Encoding.GetEncoding` below).

Its anyways better to return `null` because `Encoding.GetEncoding` loops through all registered providers and expects them to return null if they cannot resolve that encoding (which we cannot).